internal/application/application.go

```diff
@@ -322,10 +322,7 @@ func List(ctx context.Context, cluster *kubernetes.Cluster, namespace string) (m
 
 	// V. Pod metrics and replica information
 
-	metrics, err := GetPodMetrics(ctx, cluster, namespace)
-	if err != nil {
-		return nil, err
-	}
+	metrics, _ := GetPodMetrics(ctx, cluster, namespace)
```

Comment:
I would not ignore completely the error, let's log it at least.

Reply:
Done, commit 08057f81b20e3de68c33f79b6f4055a66e2fcbb9